content/tracing/setup/first_class_dimensions.md

```diff
@@ -0,0 +1,151 @@
+---
+title: First-Class Dimensions
+kind: documentation
+aliases:
+  - /tracing/environments/
+  - /tracing/setup/environment
+further_reading:
+- link: "tracing/setup/docker"
+  tag: "Documentation"
+  text: Docker setup
+- link: "tracing/setup/go"
+  tag: "Documentation"
+  text: Go language instrumentation
+- link: "tracing/setup/java"
+  tag: "Documentation"
+  text: Java language instrumentation
+- link: "tracing/setup/python"
+  tag: "Documentation"
+  text: Python language instrumentation
+- link: "tracing/setup/ruby"
+  tag: "Documentation"
+  text: Ruby language instrumentation
+---
+
+There are several dimensions that you can configure to scope an entire Datadog APM application. This includes aggregate statistics (such as requests/second, latency, error rate, Apdex score) and visible traces.
+
+## Environment
+### Definition
+
+An environment is a first-class dimension that is used to scope an entire Datadog APM application. Some display settings are shared across environments, but all the measurable data (traces/metrics/statistics) can not be re-aggregated across multiple environments. Use cases include:
+
+* Stage environments such as production, staging, and pre-production...
+
+Environments are [tags][1], therefore they must follow the following rules:
+
+* They must start with a lower case letter.
+* Other characters must be alphanumeric lower case Unicode characters, underscores, minuses, colons, periods or slashes.
+* They must not be more than 100 characters long.
+
+Environments in traces and configuration files are normalized:
+
+* Unsupported characters are replaced by underscores.
+* Upper case characters are converted to lower case.
+
```

Comment:
It might also be better just to link to the tags best practices page rather than just repeating some of the tag name rules.

Reply:
@lejulie @derekhowles do we have more restrictions on Primary tags than on usual tags?